src/lib/Tray.ts

```diff
@@ -19,56 +20,30 @@ const INDICATOR_TRAY_INDIRECT = 'tray-indirect';
 
 // TODO: Need to support i18n for a lot of the hard-coded strings in this file
 export default class TrayIcon {
-  trayIcon = null;
+  trayIcon: Tray | null = null;
```

Comment:
Should this variable name be changed to `tray` to reflect the correct type?

Reply:
yes,
I didn't do it in current commit as it have furthermore testing.
I can do that in next PR.